routing/routing_integration_tests/routing_test_tools.cpp

```diff
@@ -216,6 +217,16 @@ namespace integration
         ("Route time test failed. Expected:", expectedRouteSeconds, "have:", routeSeconds, "delta:", delta));
   }
 
+  void TestRoutePointsNumber(Route const & route, size_t expectedPointsNumber, double relativeError)
+  {
+    CHECK(relativeError > 0.0 && relativeError < 1.0, ());
```

Comment:
It would be nice to display relativeError in the diagnostic message.

Reply:
Why it can't be zero? (for exact match)

Reply:
It can. And it can be > 1. Fixed.
I changed to CHECK_GREATER_OR_EQUAL here so relativeError will be in diagnostic message.